data/json/items/ammo_types.json

```diff
@@ -242,37 +242,37 @@
   {
     "type": "ammunition_type",
     "id": "500",
-    "name": ".500 Magnum",
+    "name": ".500 S&W Magnum",
     "default": "500_Magnum"
   },
   {
     "type": "ammunition_type",
     "id": "57",
-    "name": "5.7mm",
+    "name": "5.7x28mm",
     "default": "57mm"
   },
   {
     "type": "ammunition_type",
     "id": "46",
-    "name": "4.6mm",
+    "name": "4.6x30mm",
     "default": "46mm"
   },
   {
     "type": "ammunition_type",
     "id": "762",
-    "name": "7.62x39mm",
+    "name": "7.62x39mm Soviet",
     "default": "762_m87"
   },
   {
     "type": "ammunition_type",
     "id": "545x39",
-    "name": "5.45x39mm",
+    "name": "5.45x39mm Soviet",
     "default": "545"
   },
   {
     "type": "ammunition_type",
     "id": "223",
-    "name": ".223",
+    "name": ".223/5.56x45mm NATO",
```

Comment:
I've brought this up before, it gets a "no". .223 and 556 aren't fully 100% interchangeable in some areas.